<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEditDialog</class>
 <widget class="QDialog" name="BookEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin" stdset="0">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="back_btn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>189</width>
     <height>187</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="type_label">
      <property name="text">
       <string>类型</string>
      </property>
      <property name="buddy">
       <cstring>type_combo_box</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="2">
     <widget class="QLabel" name="price_label">
      <property name="text">
       <string>价格</string>
      </property>
      <property name="buddy">
       <cstring>price_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="auther_edit"/>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="num_label">
      <property name="text">
       <string>现有数量</string>
      </property>
      <property name="buddy">
       <cstring>num_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="auther_label">
      <property name="text">
       <string>作者</string>
      </property>
      <property name="buddy">
       <cstring>auther_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="2" rowspan="2">
     <widget class="QLineEdit" name="press_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="press_label">
      <property name="text">
       <string>出版社</string>
      </property>
      <property name="buddy">
       <cstring>press_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="price_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="type_combo_box">
      <item>
       <property name="text">
        <string>教育</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>童话</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>文学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>科幻</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="name_edit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>书名</string>
      </property>
      <property name="buddy">
       <cstring>name_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="num_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="id_label">
      <property name="text">
       <string>书籍id</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="id_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>BookEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>BookEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
